common/oatbox/filesystem/FileSystemHandler.php

```diff
@@ -126,4 +128,14 @@ protected function sanitizePath($path)
 
         return $path;
     }
+
+    public function __sleep()
+    {
+        return ['fileSystemId', 'prefix'];
+    }
```

Comment:
This will probably break the descendent classes, adding more of their own properties.

WDYT of adding a constant like `private const UNSERIALIZABLE_PROPERTIES = ['fileSystem']`, and doing something like this?

```suggestion
    public function __sleep()
    {
        return array_diff($this->getAllProperties(), self::UNSERIALIZABLE_PROPERTIES);
    }
    
    private function getAllProperties(): array
    {
        return array_map(
            static function (ReflectionProperty $property): string {
                return $property->getName();
            },
            (new ReflectionClass($this))->getProperties()
        );
    }
```

Reply:
good point